src/renderer/helpers/api/local.js

```diff
@@ -580,6 +580,18 @@ function parseListItem(item) {
         descriptionShort: channel.description_snippet.text
       }
     }
+    case 'HashtagTile': {
+      /** @type {import('youtubei.js').YTNodes.HashtagTile} */
+      const hashtag = item
+
+      return {
+        type: 'hashtag',
+        title: hashtag.hashtag.text,
+        videoCount: hashtag.hashtag_video_count.isEmpty() ? null : parseLocalSubscriberCount(hashtag.hashtag_video_count.text),
+        channelCount: hashtag.hashtag_channel_count.isEmpty() ? null : parseLocalSubscriberCount(hashtag.hashtag_channel_count.text),
+        url: hashtag.endpoint.metadata.url
```

Comment:
Is there a case where the URL would be NOT `/hashtag/`?
Need a test case for that

If not I have no idea why url is needed

Reply:
@ChunkyProgrammer